framework/include/cppmicroservices/BundleContext.h

```diff
@@ -343,23 +346,22 @@ class US_Framework_EXPORT BundleContext
 
   /**
    * Returns a list of <code>ServiceReference</code> objects. The returned
-   * list contains services that
-   * were registered under the specified class and match the specified filter
-   * expression.
+   * list contains services that were registered under the specified class
+   * and match the specified filter expression.
    *
    * <p>
-   * The list is valid at the time of the call to this method. However since
+   * The list is valid at the time of the call to this method. However, since
    * the Micro Services framework is a very dynamic environment, services can be modified or
```

Comment:
There's an inconsistency. A previous usage of ```Micro Services framework``` was replaced with ```framework```. This usage probably needs to be renamed appropriately.